app/src/components/templateEditor/TemplateEditor.tsx

```diff
@@ -59,3 +60,30 @@ export const TemplateEditor = ({
     />
   );
 };
+
+export const TemplateEditorWrap = ({
+  children,
+}: {
+  children: React.ReactNode;
+}) => {
+  return (
+    <div
+      css={css`
+        & .cm-content {
+          padding: var(--ac-global-dimension-size-100)
+            var(--ac-global-dimension-size-250);
+        }
+        & .cm-gutter,
```

Comment:
Suggestion - I think we should remove the background color and the flickering mouse cursor in read-only mode - it looks really editable otherwise.

<img width="1201" alt="Screenshot 2024-12-19 at 1 26 51 PM" src="https://github.com/user-attachments/assets/d7298d9b-ab0e-4d5f-a648-daa89b9ccbe7" />

Reply:
Done